modules/features/podcasts/src/main/java/au/com/shiftyjelly/pocketcasts/podcasts/view/share/ShareListCreateViewModel.kt

```diff
@@ -99,7 +99,7 @@ class ShareListCreateViewModel @Inject constructor(
                     type = "text/plain"
                     putExtra(Intent.EXTRA_TEXT, url)
                 }
-                startActivity(context, Intent.createChooser(intent, label), null)
+                context.startActivity(Intent.createChooser(intent, label), null)
```

Comment:
Praise (❤️): Although this wasn't breaking the build, nevertheless, thanks for making this change too. 🥇